internal/runners/prepare/prepare_windows.go

```diff
@@ -0,0 +1,115 @@
+package prepare
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"github.com/gobuffalo/packr"
+
+	"github.com/ActiveState/cli/cmd/state-tray/pkg/autostart"
+	"github.com/ActiveState/cli/internal/appinfo"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/osutils"
+	"github.com/ActiveState/cli/internal/osutils/shortcut"
+	"github.com/ActiveState/cli/internal/rtutils"
+)
+
+func (r *Prepare) prepareOS() error {
+	err := setStateProtocol()
+	if err != nil {
+		r.reportError(locale.T("prepare_protocol_warning"), err)
+	}
+
+	if !rtutils.BuiltViaCI { // disabled while we're still testing this functionality
+		if err := autostart.New().Enable(); err != nil {
+			r.reportError(locale.Tr("err_prepare_autostart", "Could not enable auto-start, error received: {{.V0}}.", err.Error()), err)
+		}
+
+		if err := prepareStartShortcut(); err != nil {
+			r.reportError(locale.Tr("err_prepare_shortcut", "Could not create start menu shortcut, error received: {{.V0}}.", err.Error()), err)
+		}
+	}
+
+	return nil
+}
+
+func prepareStartShortcut() error {
+	dir := filepath.Join(os.Getenv("USERPROFILE"), "AppData", "Roaming", "Microsoft", "Windows", "Start Menu", "Programs", "ActiveState")
```

Comment:
Nit: you can use `APPDATA` here to get directly to the `Roaming` directory so:
```suggestion
	dir := filepath.Join(os.Getenv("APPDATA"), "Microsoft", "Windows", "Start Menu", "Programs", "ActiveState")
```

Can also use `APPDATA` in `autostart_windows.go`

Reply:
I'm a little concerned with using `APPDATA` as there's some mixed messages around its availability and value, eg. https://stackoverflow.com/questions/37946282/why-does-appdata-in-windows-7-seemingly-points-to-wrong-folder